build/tenuki.js

```diff
@@ -550,7 +550,7 @@ var GameState = function GameState(_ref) {
 
 GameState.prototype = {
   _nextColor: function _nextColor() {
-    if (!this.color || this.color == "white") {
+    if (this.color == "white") {
```

Comment:
This is another sign that the current naming setup is confusing. Calling `game.currentMove()` before any real moves have been made will give you meaningful board state to avoid checking `!game.currentMove()` all over the place. But with handicap stones, the first player is now no longer just black, and the way this is implemented here is to check `.color`, which is misleading: the initial state _isn't_ a move, so `game.currentMove()` shouldn't really be a legit value.

I think my preference here is to first move all naming around `currentMove()` over to something like `currentBoardState()`, and change `GameState` to `BoardState`. Concern over getting the actually played moves is then a question of inspecting _board state_, which is a lower-level issue.